feathr_project/feathr/settings.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+from jinja2 import Template
+
+
+class ObservationSettings:
+    """Time settings of the observation data. Used in feature join.
+    Attributes:
+        observation_path: path to the observation dataset, i.e. input dataset to get with features
+        output_path: output path of job, i.e. the observation data with features attached.
+        event_timestamp_column: column name of the event timestamp
+        timestamp_format: the format of the event_timestamp_column, e.g. yyyy/MM/DD.
+    """
+    def __init__(self,
+                 observation_path: str,
+                 output_path: str,
```

Comment:
output path should not be part of the observationSettings?

It's hard to explain in documentation as well.